cernopendata/modules/theme/assets/semantic-ui/js/search/components/Facets.js

```diff
@@ -39,9 +40,13 @@ const CODFacets = ({ aggs, updateQueryState, currentQueryState }) => {
         <div class="content"><ActiveFilters /></div>
       </div>
 
-      {aggs.map((agg) => (
-        <BucketAggregation key={agg.title} title={agg.title} agg={agg.agg} />
-      ))}
+      {aggs.map((agg) =>
+        agg.agg.aggName === 'year' ? (
```

Comment:
I was wondering: instead of doing this based on the name of the aggregation, would it be possible to make it based on the type of aggregation (term_filter or range_filter)? If so, we would also get this new facet for the number of events, which would be cool